fix "undefined reference to main" error when building sciphi_test
cfr. https://github.com/cbg-ethz/SCIPhI/issues/12 and https://github.com/cbg-ethz/SCIPhI/commit/1ae0b4ccad76cf67a7a074f611f7d64af568c561
--- SCIPhI-0.1.3/test/sciphi_test.cpp.orig	2019-01-14 17:03:49.121631226 +0100
+++ SCIPhI-0.1.3/test/sciphi_test.cpp	2019-01-14 17:03:57.311645226 +0100
@@ -1,4 +1,6 @@
 #define BOOST_TEST_MODULE ScateTest
+#define BOOST_TEST_DYN_LINK
+
 #include <boost/test/unit_test.hpp>
 
 #include "src/sciphi_config.h"
